payroll/serializers.py

```diff
@@ -141,17 +141,23 @@ def employer_payroll(self):
     def get_salaries(self, obj):
         data = []
 
-        # Sorting is expensive! Use self.employer_salaries as a subquery so we
-        # are only ordering salaries from the given employer and year.
-        top_salaries = Salary.objects.filter(
-            id__in=self.employer_salaries.values_list('id', flat=True)
-        ).order_by('-total_pay')[:5]
+        with connection.cursor() as cursor:
```

Comment:
if we had the field mappings we could interact with this through the ORM, which would be good.